zerver/views/home.py

```diff
@@ -253,9 +253,15 @@ def home_real(request: HttpRequest) -> HttpResponse:
         page_params['translation_data'] = get_language_translation_data(request_language)
 
     csp_nonce = generate_random_token(48)
+    emojiset = user_profile.emojiset
+    if emojiset == UserProfile.TEXT_EMOJISET:   # nocoverage
```

Comment:
We can add coverage for this; I think just a tweak to `test_home.py` to have one of the existing tests set the emojiset to text would do.